aiohttp/web_protocol.py

```diff
@@ -144,6 +144,10 @@ class RequestHandler(BaseProtocol):
 
     max_headers -- Optional maximum header size
 
+    timeout_ceil_threshold -- Optional threshold value
```

Comment:
`Optional value to trigger ceiling of timeout values`

Maybe also a link to documentation about the purpose of ceiling.

Should also update the note in the documentation to mention that the value is configurable.

Reply:
Done in the last commit